newIDE/app/src/AssetStore/AssetStoreSearchFilter.js

```diff
@@ -94,6 +94,10 @@ export class DimensionAssetStoreSearchFilter
   }
 
   getPertinence(searchItem: AssetShortHeader): number {
+    // Assets with no defined dimensions are always pertinent. (ex: particle emitter)
+    if (!searchItem.width && !searchItem.height) {
```

Comment:
Should that be:

```suggestion
    if (!searchItem.width || !searchItem.height) {
```

for safety?